eva_submission/eload_validation.py

```diff
@@ -117,12 +126,24 @@ def parse_vcf_check_report(self, vcf_check_report):
                         error_list.append(line.strip())
         return valid, error_list, error_count, warning_count
 
+    def _generate_csv_mappings(self):
+        vcf_files_csv = os.path.join(self.eload_dir, 'vcf_files.csv')
+        with open(vcf_files_csv, 'w', newline='') as file:
+            writer = csv.writer(file)
+            writer.writerow(['vcf', 'fasta', 'report'])
+            analyses = self.eload_cfg.query('submission', 'analyses')
+            for analysis_alias in analyses:
+                fasta = self.eload_cfg.query('submission', 'analyses', analysis_alias, 'assembly_fasta')
+                report = self.eload_cfg.query('submission', 'analyses', analysis_alias, 'assembly_report')
+                for vcf_file in self.eload_cfg.query('submission', 'analyses', analysis_alias, 'vcf_files'):
+                    writer.writerow([vcf_file, fasta, report])
```

Comment:
Sorry I wasn't clear in my last comment, I was thinking something like this to help clean up some of the `self.eload_cfg.query(...)`s
```suggestion
            for analysis_alias, analysis_data in analyses.items():
                fasta = analysis_data['assembly_fasta']
                report = analysis_data['assembly_report']
                for vcf_file in analysis_data['vcf_files']:
                    writer.writerow([vcf_file, fasta, report])
```